src/languages/lsl.js

```diff
@@ -0,0 +1,90 @@
+/*
+Language: Linden Scripting Language
+Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+Author: Builder's Brewery <buildersbrewery@gmail.com>
+Category: scripting
+*/
+
+function(hljs) {
+
+    var LSL_STRING_ESCAPE_CHARS = {
+        className: 'subst',
+        begin: '\\[\\\\nt"\\]'
+    };
+
+    var LSL_STRINGS = {
+        className: 'string',
+        begin: '"',
+        end: '"',
+        contains: [
+            { begin: '/\\*', end: '\\*/', skip: true },
```

Comment:
No special meaning. Was trying to make sure strings would override comments contained within.